Userland/Libraries/LibWeb/HTML/BrowsingContext.cpp

```diff
@@ -321,8 +320,10 @@ JS::NonnullGCPtr<HTML::TraversableNavigable> BrowsingContext::top_level_traversa
 // https://html.spec.whatwg.org/multipage/browsers.html#top-level-browsing-context
 bool BrowsingContext::is_top_level() const
 {
-    // A browsing context that has no parent browsing context is the top-level browsing context for itself and all of the browsing contexts for which it is an ancestor browsing context.
-    return !parent();
+    // FIXME: Remove this. The active document's navigable is sometimes null when it shouldn't be, failing assertions.
+    return true;
```

Comment:
hm, making is_top_level() always return true is more likely to cause more problems than it solves. could we instead make all callers check if browsing context have a navigable?

Reply:
I need to look into it more, but from what I remember there are places where the spec asserts it to not be null, so adding checks there would create more fixmes/hacks. Like I said this does not change behaviour from current `master`, because it was already always returning `true`, the proper way to solve this would be making sure navigable isn't `null` when it shouldn't be.